rest/rest-api/src/main/java/org/ow2/proactive_grid_cloud_portal/common/SchedulerRestInterface.java

```diff
@@ -2160,6 +2160,7 @@ List<FilteredTopWorkflowsCumulatedCoreTimeData> getTopWorkflowsCumulatedCoreTime
      * @param startDate start date of the filtered jobs
      * @param endDate end date of the filtered jobs
      * @param workflowName the workflow name of the filtered jobs
+     * @param inParallel fetch only the jobs that use the most nodes in parallel
```

Comment:
if true, returns the maximum number of nodes used in parallel instead of the total number of nodes.